View/TableWithPlot/FdaDataGrid.cs

```diff
@@ -293,13 +292,15 @@ public void PasteClipboard()
                             Binding binding = (Columns[ColumnIndex + j] as DataGridBoundColumn).Binding as Binding;
                             Type RowType = Items[RowIndex + i].GetType();
                             PropertyInfo y = RowType.GetProperty(binding.Path.Path);
-                            if(ColumnIndex+j == Columns.Count-1)
+
+                            try
                             {
-                                y.SetValue(Items[RowIndex + i], Convert.ChangeType(clipboardData[i][0], y.PropertyType));
+                                var newVal = Convert.ChangeType(clipboardData[i][j], y.PropertyType);
+                                ((SequentialRow)Items[RowIndex + i]).UpdateRow(ColumnIndex + j, (double)newVal);
                             }
-                            else
+                            catch (Exception ex)
                             {
-                                y.SetValue(Items[RowIndex + i], Convert.ChangeType(clipboardData[i][j + 1], y.PropertyType));
+                                //ignore
```

Comment:
Are the potential exceptions not worth handling?

Reply:
Recycled code in here. Looks like the original author didn't think so. I'd be ok accepting it as is for now until we find an issue with it.